source/includes/api/_contacts.md

```diff
@@ -1,677 +1,852 @@
 # Contacts
 
-## The contact object
-
-```json
-{
-  "id": "0d011563-c7c6-4a09-8fde-f3b8c6f2b22d",
-  "customerId": "e3899af9-2b74-45dd-b2b1-d7be8ad47bae",
-  "securityGroupId": "d4a1d326-a2c7-415c-bfea-dd9fc84f8d00",
-  "data": {
-    "listId": "2210d9ea-1870-4b8d-8803-ce426e33a96b",
-    "profile": {
-      "name": {
-        "first": "John",
-        "last": "Snow",
-        "title": "Mr"
-      },
-      "address": {
-        "country": "United States of America",
-        "position": {
-          "type": "Point",
-          "coordinates": [
-            -71.067513,
-            42.3520436
-          ]
-        },
-        "formatted": "100 Boylseton Street, Boston, MA 02266, USA",
-        "streetAddress": {
-          "formatted": "100 Boylseton Street",
-          "number": "100",
-          "route": "Boylseton Street"
-        },
-        "postalCode": "02266",
-        "state": "Massachusettes",
-        "county": "Suffolk County",
-        "city": "Boston",
-        "neighborhood": "Downtown Boston"
-      },
-      "email": "john@testemail.com",
-      "phone": [
-        "9781234568",
-        "3171234568",
-        "2631234568"
-      ],
-      "gender": "m",
-      "languages": [
-        "en"
-      ],
-      "birthdate": "1987-09-03",
-      "age": 30
-    },
-    "notes": "test notes",
-    "fields": {
-      "available": "true",
-      "bestpet": "Cat",
-      "buypet": [
-        "Horse",
-        "Snake"
-      ],
-      "mothername": "Jane Doe"
-    },
-    "integration": {
-      "integrationId": "e5f5b8b9-9275-426d-ae52-3e684b6c7c76",
-      "externalId": 1001,
-    },
-    "reservation": {
-      "reserved": false
-    },
-    "contacted": true
-  },
-  "meta": {
-    "created": "2017-12-15T19:46:01.594Z",
-    "modified": "2017-12-15T19:46:02.219Z",
-    "isDeleted": false,
-    "resource": "contacts",
-    "createdBy": "11b78eab-8b3e-45e7-804c-4a94045367d4",
-    "modifiedBy": "11b78eab-8b3e-45e7-804c-4a94045367d4",
-    "etag": "MTUxMzM2NzE2NTI3MzM3NDcyMA=="
-  }
-}
-```
+## The Contacts object
 
 Attribute | Required? | Description
 --------- | --------- | -----------
-id | true | Unique identifier for the object.
-customerId | true | Customer this resource belongs to.
-securityGroupId | true | Security group of this resource.
-data | true | Contact data.
+id | true | Identifier of this record
+securityGroupId | true | Identifier of the associated securitygroup
+customerId | true | Identifier of the associated customer
+data | true | Todo: description
 
 ### data
 
 Attribute | Required? | Description
 --------- | --------- | -----------
-listId | true | List identifier this contact belongs to.
-profile | true | Contact profile [details](#data-profile).
+profile | true | Contact profile
```

Comment:
ok look at profile and see the link to the profiles section, can you add this? the best way to do this is just to manually change the keys on the original contacts document

Reply:
Okay ..on it